### PR TITLE
belindas-closet-nextjs_8_429_create-all-products-page

### LINKED ISSUE #429: Create “All Products” page that displays all the products from the database
Tasks:

- [x] Create “All Products” page as  /category-page/all-products
- [x] Set up similar to other pages, but display all the products from the database

### PR DESCRIPTION
**Pull request by heosman**
Resolves #429 

This PR creates an All Products page at http://localhost:8080/category-page/all-products that displays all active products from the database that are not deleted or archived.

image

### CODE REVIEW

**Comment by kaimanasse15** [comment]
Hi @heosman , the page that you created look good with the link but I am wordering how to see the products have been added? Thank you.
image

**Comment by keiffer213** [comment]
@kaimanasse15 It works if you have products in your local database, but by changing the URL in the app-products page just like the 2 photos below, you can connect to the production database

@heosman should we update the URL to connect to the belinda's closet API?

URL In all-products page.tsx
image

image

image

**Comment by heosman** [comment]
> @kaimanasse15 It works if you have products in your local database, but by changing the URL in the app-products page just like the 2 photos below, you can connect to the production database
> 
> @heosman should we update the URL to connect to the belinda's closet API?
> 
> URL In all-products page.tsx image
> 
> image
> 
> image

I forgot to add the Belinda's Closet API back to the URL when I was testing, but it should be there now.

**Comment by MuhammadNSC** [comment]
I'm also no seeing any product and i think its because i dont have any products in my local database, but this should be a good change to belinda closet.
Screenshot 2024-06-05 at 10 18 09 PM

**Review by keiffer213 — APPROVED** [review]
Looks great! Thanks for this!

File: app/category-page/all-products/page.tsx
Comment (gitbiruk2010) [inline app/category-page/all-products/page.tsx:1]: Good job @heosman! Products page looks clean and nice.

**Review by Asfand00 — APPROVED** [review]
looks good to me, nice work @heosman.
all-products-page

**Comment by Seiyaroo** [comment]
Looks great!

**Comment by tinpham5614** [comment]
For better responsiveness, you can use the grid styling in line 77 instead of line 78.
image
The result will look like this:
image

**Pull request merged**